packages/web-app-user-management/src/components/AppTemplate.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <main class="oc-flex oc-height-1-1 app-content oc-width-1-1">
+    <app-loading-spinner v-if="loading" />
+    <template v-else>
+      <div id="user-management-wrapper" class="oc-width-expand">
+        <div id="user-management-app-bar" ref="appBar" class="oc-app-bar oc-py-s">
+          <div class="oc-flex oc-flex-between">
+            <oc-breadcrumb class="oc-flex oc-flex-middle" :items="breadcrumbs" />
+            <div>
+              <oc-button
+                id="files-toggle-sidebar"
+                v-oc-tooltip="toggleSidebarButtonLabel"
+                :aria-label="toggleSidebarButtonLabel"
+                appearance="raw"
+                class="oc-my-s oc-p-xs"
+                @click.stop="$emit('toggleSideBar')"
+              >
+                <oc-icon name="side-bar-right" :fill-type="toggleSidebarButtonIconFillType" />
+              </oc-button>
+            </div>
+          </div>
+          <slot name="topbarActions" class="oc-flex-1 oc-flex oc-flex-start" />
+        </div>
+        <slot name="mainContent" />
+      </div>
+      <side-bar
+        v-if="sideBarOpen"
+        class="user-management-side-bar oc-width-1-1 oc-width-1-3@m oc-width-1-4@xl"
```

Comment:
Sidebar in the files app has a fixed width by now (except for mobile resolutions, where it takes 100% of the width). Worth to apply that here as well?

Reply:
Needs a little code base changes, but done